telegram/user.py

```diff
@@ -233,6 +234,24 @@ def mention_html(self, name: str = None) -> str:
             return util_mention_html(self.id, name)
         return util_mention_html(self.id, self.full_name)
 
+    def mention_button(self, name: str = None) -> InlineKeyboardButton:
+        """
+        Shortcut for::
+
+            InlineKeyboardButton(text=name, url=f"tg://user?id={update.effective_user.id}")
+
+        .. versionadded:: 13.9
+
+        Args:
+            name (:obj:`str`): The name used as a link for the user. Defaults to :attr:`full_name`.
+
+        Returns:
+            :obj:`InlineKeyboardButton`: InlineButton with url set to the user mention
+        """
+        if name:
+            return InlineKeyboardButton(text=name, url=f"tg://user?id={self.id}")
+        return InlineKeyboardButton(text=self.full_name, url=f"tg://user?id={self.id}")
```

Comment:
both `mention_html` and `mention_markdown` have similar f-strings for the link. Is it worth extracting that to another helper function?

Reply:
I dont think so, not for V13. We can discuss this for V14 imo.